src/ansys/geometry/core/geometry/surfaces/cone.py

```diff
@@ -235,37 +240,35 @@ def project_point(self, point: Point3D) -> "ConeEvaluation":
 
         return ConeEvaluation(self, ParamUV(u, v))
 
-    def get_u_parameterization(self) -> Parameterization:
+    def parameterization(self) -> tuple[Parameterization, Parameterization]:
         """
-        Get the parametrization conditions for the U parameter.
+        Parameterization of the cone surface as a tuple (U and V respectively).
```

Comment:
```suggestion
        Parameterize the cone surface as a tuple (U and V respectively).
```
Earlier, ``u`` and ``v`` in lower case letters were used and tagged as code entities. We should be consistent. Please decide which to use and do a search and replace so that they are consistent throughout.